modules/auxiliary/scanner/oracle/oracle_hashdump.rb

```diff
@@ -25,22 +26,37 @@ def initialize
   def run_host(ip)
     return if not check_dependencies
 
-    # Checks for Version of Oracle, 8g-10g all behave one way, while 11g behaves differently
-    # Also, 11g uses SHA-1 while 8g-10g use DES
-    is_11g=false
+    # Checks for Version of Oracle. Behavior varies with oracle version.
+    # 12c uses SHA-512 (explained in more detail in report_hashes() below)
+    # 11g uses SHA-1 while 8i-10g use DES
     query =  'select * from v$version'
     ver = prepare_exec(query)
 
     if ver.nil?
-      print_error("An Error has occurred, check your OPTIONS")
+        print_error("An Error has occurred. Check your OPTIONS")
```

Comment:
why the 2 space indent?
Also, lets make this more informative on *where* in the process it failed.  Maybe something like "An error has occurred while querying for the Oracle version.  Please check your OPTIONS"